src/test/java/it/pagopa/pn/bff/pnclient/auth/PnAuthFleetClientImplTest.java

```diff
@@ -0,0 +1,95 @@
+package it.pagopa.pn.bff.pnclient.auth;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import it.pagopa.pn.bff.mocks.AuthFleetMock;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockserver.client.MockServerClient;
+import org.mockserver.integration.ClientAndServer;
+import org.mockserver.model.MediaType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import reactor.test.StepVerifier;
+
+import static org.mockserver.integration.ClientAndServer.startClientAndServer;
+import static org.mockserver.model.HttpRequest.request;
+import static org.mockserver.model.HttpResponse.response;
+
+@SpringBootTest(properties = {
+        "pn.bff.auth-fleet-base-url=http://localhost:9998",
+        "cors.allowed.domains=http://localhost:8080"
+})
+public class PnAuthFleetClientImplTest {
```

Comment:
This is the integration test. so rename it with PnAuthFleetClientImplTestIT and add the PnAuthFleetClientImplTest file